sdk/storage/azblob/internal/generated/autorest.md

```diff
@@ -321,3 +321,14 @@ directive:
     from: BlobPropertiesInternal
     to: BlobProperties
 ```
+
+### Updating encoding URL, Golang adds '+' which disrupts encoding with service
+
+``` yaml
+directive:
+  - from: zz_service_client.go
+    where: $
+    transform: >-
+      return $.
+        replace(/req.Raw\(\).URL.RawQuery \= reqQP.Encode\(\)/, `req.Raw().URL.RawQuery = strings.Replace(reqQP.Encode(), "+", "%20", -1)`)
```

Comment:
I don't see any changes to the generated code.  Did you run `go generate` from the `generated` directory after making this change?  Also, shouldn't the tests be failing without the fix being applied?